test/controllers/admin/items_in_maintenance_controller_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+class Admin::ItemInMaintenanceControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    @user = create(:admin_user)
+    sign_in @user
+  end
+
+  test "should get index" do
+    get admin_items_in_maintenance_index_url
```

Comment:
If we create an item before making the request, the view will render the table exercise that code as well. There are factories for most of the models in `test/factories`, including one for Item:

```suggestion
    create(:complete_item, status: Item.statuses[:maintenance])
    get admin_items_in_maintenance_index_url
```